gatsby_fp/src/components/fieldComponents/IntellCont.js

```diff
@@ -1,112 +1,160 @@
 import React from "react"
-const shortid = require("shortid")
+import SubHeading from "./intellContComponents/SubHeading.js"
```

Comment:
This looks like a mix of camelCase and PascalCase. Do we need the mix?

Reply:
No, I'll rewrite the filenames to pascal case as well